src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -375,21 +376,10 @@ const WorkflowView = _.flow(
   updateSingleOrMultipleRadioState(config) {
     this.setState({
       processSingle: !config.rootEntityType,
-      selectedEntityType: config.rootEntityType
+      selectedEntityType: config.dataReferenceName ? config.dataReferenceName : config.rootEntityType
```

Comment:
This could actually be simplified to:
```suggestion
      selectedEntityType: config.dataReferenceName || config.rootEntityType
```